build.sbt

```diff
@@ -86,12 +86,21 @@ publishTo := {
 
 pomIncludeRepository := { _ => false }
 
-pomExtra :=
-  <url>https://github.com/slicebox/dicom-streams</url>
-    <developers>
-      <developer>
-        <id>KarlSjostrand</id>
-        <name>Karl Sjöstrand</name>
-        <url>https://github.com/KarlSjostrand</url>
-      </developer>
-    </developers>
\ No newline at end of file
+scmInfo := Some(
+  ScmInfo(
+    url("https://github.com/exini/dicom-streams"),
+    "scm:git@github.com:exini/dicom-streams.git"
+  )
+)
+
+homepage := Some(url("https://github.com/exini/dicom-streams"))
+
+
+developers := List(
+  Developer(
+    id    = "KarlSjostrand",
```

Comment:
My github username is karl-exini. Is that what should go here?

Reply:
fixed